lib/babel-istanbul-instrumenter.js

```diff
@@ -0,0 +1,256 @@
+/**
```

Comment:
@sophypal thanks for pointing that out. I can't use this yet because of my use of `ember-computed-decorators` in every app I work on. But I agree the value add for ES6 is definitely there.